Misc/NEWS.d/next/C API/2021-04-16-18-15-56.bpo-43868.twQ7KH.rst

```diff
@@ -0,0 +1,3 @@
+:c:func:`PyOS_ReadlineFunctionPointer` is no longer exported by
```

Comment:
Do you think that it's worth it to be mentioned in C API Changes of What's New in Python 3.10?
https://docs.python.org/dev/whatsnew/3.10.html#id2 ?

Reply:
Yes, that sounds reasonable. Thanks for the catch!